tests/ui/functions_maxlines.rs

```diff
@@ -1,162 +0,0 @@
-#![warn(clippy::all, clippy::pedantic)]
```

Comment:
Oh you could have left this test in, so we know that it also works with the default configuration.

Reply:
Fair! I can add it back. I was just thinking about keeping everything in one place, but this makes sense.